examples/docs_snippets/docs_snippets/guides/dagster/development_to_production/branch_deployments/clone_prod.yaml

```diff
@@ -22,7 +22,8 @@ name: Dagster Branch Deployments
           with:
             location: ${{ toJson(matrix.location) }}
             deployment: ${{ steps.deploy.outputs.deployment }}
-            repository: repo
+            # TODO need to change github action to default to __repository__
+            repository: __repository__
```

Comment:
is this blocked by @benpankow 's changes in cloud? if so, could we make it clear in the summary as a todo?